custom_components/unifiprotect/services.py

```diff
@@ -22,65 +24,101 @@ def _async_all_ufp_instances(hass: HomeAssistant) -> list[ProtectApiClient]:
     ]
 
 
+def _async_get_macs_for_device(device_entry: dr.DeviceEntry) -> list[str]:
+    return [
+        # MAC addresses in UFP are always caps
+        cval.replace(":", "").upper()
```

Comment:
use dr.format_mac

Reply:
This is actually to revert the formatting of the mac so it matches the UFP value. The addresses for UFP are always capital with no `:`. 

![image](https://user-images.githubusercontent.com/490848/144723340-4dfa4dc3-a4c8-4a57-99bd-a0c09e93c42d.png)

Reply:
Or do it the other way around and run `i.bootstrap.nvr.mac` though `dr.format_mac`